src/tests/integration/test_nycdb.py

```diff
@@ -125,6 +125,22 @@ def test_hpd_complaints(conn):
     assert row_count(conn, 'hpd_complaints') == 100
 
 
+def test_dof_exemptions(conn):
+    drop_table(conn, 'dof_exemptions')
+    drop_table(conn, 'dof_exemption_classification_codes')
+    hpd_complaints = nycdb.Dataset('dof_exemptions', args=ARGS)
```

Comment:
```suggestion
    dof_exemptions = nycdb.Dataset('dof_exemptions', args=ARGS)
```
Maybe name this according to what the data is, like we've done below

Reply:
GAH thank you! I was using the hpd_complaints dataset as a template and inevitably I forgot to change a few variables...